mediatype/mediatype.go

```diff
@@ -30,16 +30,20 @@ const (
 	utf8 = "utf-8"
 )
 
+type MediaTypeParseOptions struct {
+	StripMediaTypeInvalidCharacters bool
+}
+
 // Parse is a more tolerant implementation of Go's mime.ParseMediaType function.
 //
 // Tolerances accounted for:
 //   - Missing ';' between content-type and media parameters
 //   - Repeating media parameters
 //   - Unquoted values in media parameters containing 'tspecials' characters
 //   - Newline characters
-func Parse(ctype string) (mtype string, params map[string]string, invalidParams []string, err error) {
```

Comment:
Now that enmime is post 1.0, changing the public API is a big deal.  I'd recommend renaming this function to `ParseWithOptions` or similar, and adding a wrapper `Parse` that uses default options to maintain the original API.